docs/source/installation.rst

```diff
@@ -26,7 +26,7 @@ To install the latest version tested use ``pip``:
 
 ::
 
-    pip install -r requirements.txt
+    pip install . -c requirements.txt
```

Comment:
I think we're a bit too early for that change...

Reply:
But I can revert if you want. It seemed related so I updated it.

Reply:
`pip install .` will install the tlsfuzzer package, we don't want that just yet

Reply:
Oh, I see what you mean. Though, adding `-e` (`--editable`) would probably be fine. But okay, I'll drop this for now.